core/store/migrate/migrations/0146_unique_contract_address_per_chain.sql

```diff
@@ -1,15 +1,10 @@
 -- +goose Up
-DROP INDEX IF EXISTS unique_contract_addr_per_chain;
-DROP INDEX IF EXISTS unique_contract_addr;
-CREATE FUNCTION wildcard_cmp(INTEGER, INTEGER) RETURNS INTEGER AS $$ SELECT CASE $1 = -1 OR $2 = -1 WHEN TRUE THEN 0 ELSE ($1-$2) END$$ PARALLEL SAFE IMMUTABLE LANGUAGE SQL;
-CREATE OPERATOR CLASS wildcard_cmp FOR TYPE INTEGER USING BTREE AS FUNCTION 1 wildcard_cmp (INTEGER, INTEGER), OPERATOR 1 <, OPERATOR 2 <=, OPERATOR 3 =, OPERATOR 4 >=, OPERATOR 5 >;
--- Remove all but most recently added contract_address for each chain, before creating UNIQUE constraint
-DELETE FROM ocr_oracle_specs WHERE id IN (SELECT id FROM (SELECT id, MAX(id) OVER(PARTITION BY evm_chain_id, contract_address ORDER BY id) AS max FROM ocr_oracle_specs) x WHERE id != max);
-CREATE UNIQUE INDEX ocr_oracle_specs_unique_contract_addr ON ocr_oracle_specs (COALESCE(evm_chain_id::INTEGER, -1) wildcard_cmp, contract_address);
+
+-- no-op
 
 -- +goose Down
 DROP INDEX IF EXISTS ocr_oracle_specs_unique_contract_addr;
 DROP OPERATOR CLASS IF EXISTS wildcard_cmp USING BTREE CASCADE;
```

Comment:
Did you check that this is not going to fail even if the new operator doesn't exists (I can imagine that happening when permission check is executed before IF EXISTS part).

Reply:
I tested all cases locally on my machine, but you make a good point... I should probably try creating a non-admin user and see what happens in that case.  None of us have ever tested the code under those conditions, but apparently that's how some of them have it set up.

Reply:
Looks good:
```
chainlink_test=> CREATE OPERATOR CLASS wildcard_cmp FOR TYPE INTEGER USING BTREE AS FUNCTION 1 wildcard_cmp (INTEGER, INTEGER), OPERATOR 1 <, OPERATOR 2 <=, OPERATOR 3 =, OPERATOR 4 >=, OPERATOR 5 >;
ERROR:  must be superuser to create an operator class
Time: 2.892 ms
chainlink_test=> DROP OPERATOR CLASS IF EXISTS wildcard_cmp USING BTREE CASCADE;
NOTICE:  operator class "wildcard_cmp" does not exist for access method "btree", skipping
DROP OPERATOR CLASS
Time: 2.920 ms
```